docs/docs/getting-started/first-steps.md

```diff
@@ -91,12 +63,8 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
     </tabItem>
     </tabs>
 
-    Now, fill the output values of the command into the corresponding fields of the `constellation-conf.yaml` file.
-
     :::tip
-
-    To learn how to delete your IAM configuration and to get more detailed information on the IAM process and the VM types supported by Constellation, see the [Configuration workflow](../workflows/config.md).
-
+    To learn how to create a configuration file without automatic IAM creation and how to delete your IAM configuration and to get more detailed information on the IAM process and the VM types supported by Constellation, see the [Configuration workflow](../workflows/config.md).
```

Comment:
```suggestion
    To learn about all options you have for managing IAM resources and Constellation configuration, see the [Configuration workflow](../workflows/config.md).
```
This sentence is quite long and verbose. Do you think this suggested shortening is ok or do you need more details here?